src/NimBLEAdvertisedDevice.h

```diff
@@ -95,7 +95,7 @@ class NimBLEAdvertisedDevice {
     bool m_haveTXPower;
 
 
-    NimBLEAddress  m_address = NimBLEAddress("\0\0\0\0\0\0");
+    NimBLEAddress  m_address = NimBLEAddress("\0\0\0\0\0\0", 6);
```

Comment:
This will not compile, but we can remove it and just send and empty string thanks to the other changes you made.

Reply:
No that will not compile indeed. Too quick...! I wanted to use `NimBLEAddress(std::string("\0\0\0\0\0\0", 6))` but I changed it into `NimBLEAddress("")`